docs/documentation/components/tag-input.mdx

```diff
@@ -122,20 +122,43 @@ function TagPropCallbackTagInputExample() {
   const [values, setValues] = React.useState(['Kauri', 'Willow'])
   return (
     <TagInput
-      tagProps={value => {
+      tagProps={(value) => {
```

Comment:
🤔 Is there a different `prettier` config for the `docs` site? I've seen some churn with the arrow parens + trailing commas in some other files in the repo, but I'm not quite sure where it's coming from 😅 

Not a big deal, but I'd love to get to the bottom of it 😛

Reply:
Ooo looks like there is a separate config. Didn't realize that.

Reply:
I wonder if we break out the root config from `package.json` into a `.prettierrc` file - would it use that for both `src` and `docs`? 🤔 ideally they'd match